Library/ViewModels/SearchViewModelTests.swift

```diff
@@ -779,4 +779,47 @@ internal final class SearchViewModelTests: TestCase {
       }
     }
   }
+
+  func testSearchPageViewed_BeforeSearching() {
+    self.vm.inputs.viewWillAppear(animated: true)
+
+    XCTAssertEqual(["Page Viewed"], self.dataLakeTrackingClient.events)
+    XCTAssertEqual(["Page Viewed"], self.segmentTrackingClient.events)
+
+    let dataLakeClientProps = self.dataLakeTrackingClient.properties.last
+    let segmentClientProps = self.segmentTrackingClient.properties.last
+
+    XCTAssertEqual("search", dataLakeClientProps?["context_page"] as? String)
+    XCTAssertEqual("search", segmentClientProps?["context_page"] as? String)
+
+    XCTAssertEqual("", dataLakeClientProps?["discover_search_term"] as? String)
+    XCTAssertEqual("", segmentClientProps?["discover_search_term"] as? String)
+
+    XCTAssertEqual(0, dataLakeClientProps?["discover_search_results_count"] as? Int)
+    XCTAssertEqual(0, segmentClientProps?["discover_search_results_count"] as? Int)
+  }
+
+  func testSearchPageViewed_ReturningAfterSearching() {
+    let searchProjects = (1...10).map { idx in .template |> Project.lens.id .~ idx }
```

Comment:
I think you can delete this line and on the next line do `let searchResponse = DisoveryEnvelope.template` without the rest of the line and the test will have the same desired effect.